test/integration/serverless/run.sh

```diff
@@ -157,17 +164,57 @@ trace_functions=(
     "trace-proxy"
     "otlp-python"
 )
+appsec_functions=(
+    "appsec-node"
+    "appsec-python"
+    "appsec-java"
+    "appsec-go"
+    "appsec-csharp"
+)
 
-all_functions=("${metric_functions[@]}" "${log_functions[@]}" "${trace_functions[@]}")
+declare -a all_functions # This is an array
+if [ $# == 1 ]; then
+    case $1 in
+        metric)
+            all_functions=("${metric_functions[@]}")
+        ;;
+        log)
+            all_functions=("${log_functions[@]}")
+        ;;
+        trace)
+            all_functions=("${trace_functions[@]}")
+        ;;
+        appsec)
+            all_functions=("${appsec_functions[@]}")
+        ;;
+        *)
+            echo "Unknown test suite: '$1' (valid names are: metric, log, trace, appsec)"
+            exit 1
+        ;;
+    esac
+    echo "Selected test suite '$1' contains ${#all_functions[@]} functions..."
+else
+    all_functions=("${metric_functions[@]}" "${log_functions[@]}" "${trace_functions[@]}" "${appsec_functions[@]}")
+fi
 
 # Add a function to this list to skip checking its results
 # This should only be used temporarily while we investigate and fix the test
-functions_to_skip=()
+functions_to_skip=(
+    "appsec-go" # Pending merge/release of https://github.com/DataDog/datadog-lambda-go/pull/143
```

Comment:
FYI, you could alternatively just pin the go.mod to the specific commit you need. That way you wouldn't need an extra PR.